docs/devices.md

```diff
@@ -55,79 +17,108 @@ For mice, the DPI column specifies if the mouse's sensitivity is fixed (`-`),
+  window and display the current lighting value (Lux) as reported by the
+  keyboard, similar to Logitech's *Solar.app* for Windows.
+
+* FN swap changes the way the function keys (`F1`..`F12`) work, i.e., whether holding `FN` while pressing the function keys will generate the standard `Fx` keycodes or the special function (yellow icons) keycodes.
+
+
+### Mice (Unifying):
+
+| Device           | WPID | HID++ | Battery | DPI   | Other supported features        |
+|------------------|------|-------|---------|-------|---------------------------------|
+| M150             | 4022 | 2.0   |         |       |                                 |
+| M185             | 4055 | 4.5   |         | R/W   | smooth scrolling                |
+| M310             | 4031 | 2.0   | yes     |       |                                 |
+| M310             | 4055 | 4.5   |         | R/W   | smooth scrolling                |
+| M317             |      |       |         |       |                                 |
+| M325             | 400A | 2.0   | yes     | 1000  | smooth scrolling                |
+| M345             | 4017 | 2.0   | yes     | -     | smooth scrolling                |
+| M350             | 101C | 1.0   | yes     |       |                                 |
+| M505             | 101D | 1.0   | yes     |       | smooth scrolling, side scrolling|
+| M510             | 1025 | 1.0   | yes     |       | smooth scrolling, side scrolling|
+| M510             | 4051 | 2.0   | yes     |       | smooth scrolling                |
+| M515 Couch       | 4007 | 2.0   | yes     | -     | smooth scrolling                |
+| M525             | 4013 | 2.0   | yes     | -     | smooth scrolling                |
+| M560             |      | 2.0   | yes     | -     | smooth scrolling                |
+| M585             | 406B | 4.5   | yes     | R/W   | smooth scrolling                |
+| M590             | 406B | 4.5   | yes     | R/W   | smooth scrolling                |
+| M600 Touch       | 401A | 2.0   | yes     |       |                                 |
+| M705 Marathon    | 101B | 1.0   | yes     | -     | smooth scrolling, side scrolling|
+| M705 Marathon    | 406D | 4.5   | yes     | R/W   | smooth scrolling                |
+| T400 Zone Touch  |      | 2.0   | yes     |       | smooth scrolling                |
+| T620 Touch       |      | 2.0   | yes     |       |                                 |
+| Performance MX   | 101A | 1.0   | yes     | R/W   | smooth scrolling, side scrolling|
+| Anywhere MX      | 1017 | 1.0   | yes     | R/W   | smooth scrolling, side scrolling|
+| Anywhere MX 2    | 404A | 4.5   | yes     | R/W   | smooth scrolling                |
+| MX Master        | 4041 | 4.5   | yes     | R/W   | smooth scrolling, smart shift   |
+| MX Master 25     | 4069 | 4.5   | yes     | R/W   | smooth scrolling, smart shift   |
+| Cube             |      | 2.0   | yes     |       |                                 |
```

Comment:
This is maybe a good time to drop to mention that there's no HID++ 4.5, this is just a scheme used by Logitech to report the target software. These devices use HID++ 2.0. It was a dumb decision for Logitech but we can't do anything about it. Can you change all references of HID++ 4.x to 2.0? Thanks!

Reply:
Hmm.  These devices report 4.5, which shows up in output.   How about adding a note that 4.5 is the same as 2.0?  (This doesn't disparage Logitech.)

Reply:
We should fix that too. I think we should replace the 4.x here and in the code.